kafka/src/test/resources/config/base.conf

```diff
@@ -51,27 +49,19 @@ ad-manager {
   health.status.path = "/tmp/isAlive"
 }
 
-anomaly-validator {
+// Configs for producers to kafka topic - used to send test data to topics
+producer {
   streams {
-    auto.offset.reset = "earliest"
-    cache.max.bytes.buffering = "0"
-    commit.interval.ms = "300"
-    state.dir = "/tmp/kafka-streams"
-
-    application.id = "anomaly-validator"
-    default.value.serde = "com.expedia.adaptivealerting.kafka.serde.JsonPojoSerde"
-    JsonPojoClass = "com.expedia.adaptivealerting.core.data.MappedMetricData"
-    default.timestamp.extractor = "com.expedia.adaptivealerting.kafka.serde.MappedMetricDataTimestampExtractor"
-    default.deserialization.exception.handler = "org.apache.kafka.streams.errors.LogAndContinueExceptionHandler"
+    acks = "all"
+    retries = "0"
+    key.serializer = "org.apache.kafka.common.serialization.StringSerializer"
+    value.deserializer = "com.expedia.adaptivealerting.kafka.serde.JsonPojoDeserializer"
   }
-  inbound-topic = "anomalies"
-  outbound-topic = "alerts"
+  topic = "constant-metrics"
```

Comment:
This looks outdated

Reply:
+1. I will move the whole config dir to the scala integration tests since that's what's intended. the current e2e test is ignored right now and needs a revamp. i will create a ticket for that

Reply:
https://github.com/ExpediaDotCom/adaptive-alerting/issues/260